docs/UsersGuide/source/Introduction.rst

```diff
@@ -38,6 +38,116 @@ File paths or code that include angle brackets (e.g., ``build_<platform>_<compil
 
 
+.. _SRWPrerequisites:
+
+Prerequisites for Using the SRW Application
+===============================================
+
+Background Knowledge Prerequisites
+--------------------------------------
+
+The instructions in this documentation assume that users have certain background knowledge: 
+
+* Familiarity with LINUX/UNIX systems
+* Command line basics
+* System configuration knowledge (e.g., compilers, environment variables, paths, etc.)
+* Numerical Weather Prediction
+* Meteorology (particularly meteorology at the scales being predicted)
+
+..
+   COMMENT: Suggested sub-bullets for Meteorology/NWP? Cumulus and microphysics parameterizations? Convection? Microphysics?
+
+Additional background knowledge in the following areas could be helpful:
+* High-Performance Computing (HPC) Systems for those running the SRW App on an HPC system
+* Programming (particularly Python) for those interested in contributing to the SRW App code
+* Creating an SSH Tunnel to access HPC systems from the command line
+* Containerization
+* Workflow Managers/Rocoto
+
+
+Software/Operating System Requirements
+-----------------------------------------
+The UFS SRW Application has been designed so that any sufficiently up-to-date machine with a UNIX-based operating system should be capable of running the application. NOAA `Level 1 & 2 systems <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`__ already have these prerequisites installed. However, users working on other systems must ensure that the following requirements are installed on their system: 
+
+**Minimum Platform Requirements:**
+
+* POSIX-compliant UNIX-style operating system
+
+* >40 GB disk space
+
+   * 18 GB input data from GFS, RAP, and HRRR for "out-of-the-box" SRW App case described in :numref:`Chapter %s <BuildRunSRW>`
+   * 6 GB for :term:`HPC-Stack` full installation
+   * 1 GB for ufs-srweather-app installation
+   * 11 GB for 48hr forecast on CONUS 25km domain
+
+* 4GB memory (CONUS 25km domain)
+
+* Fortran compiler with full Fortran 2008 standard support
+* Fortran compiler with support for Fortran 2003
+
+   * gfortran v9+ or ifort v18+ are the only ones tested, but others may work.
+
+..
+   COMMENT: Which one is it?! 
+
+* C and C++ compilers compatible with the Fortran compiler
+
+   * gcc v9+, ifort v18+, and clang v9+ (macOS, native Apple clang or LLVM clang) have been tested
+
+* Python v3.6+, including prerequisite packages ``jinja2``, ``pyyaml`` and ``f90nml``
+   
+   * Python packages ``scipy``, ``matplotlib``, ``pygrib``, ``cartopy``, and ``pillow`` are required for users who would like to use the provided graphics scripts
+
+* Perl 5
+
+* git v1.8+
+
+..
+   COMMENT: Should curl/wget/TIFF library also be required? These are listed as prerequisites for building HPC-Stack on generic MacOS/Linux 
```

Comment:
@willmayfield @mkavulich @kgerheiser Could one of you answer the question in line 106? Thanks!